packages/client/src/types.ts

```diff
@@ -56,33 +54,6 @@ export interface EventParams {
   [Event.PROTOCOL_MESSAGE]: [messageDetails: any, protocolName: string, sendingPeer: Peer]
 }
```

Comment:
How does the deletion of this event bus change the typing of the client types?

Reply:
`eventEmitter3` doesn't need this additional layer of type expansions and we can just use the basic `EventParams` interface keyed by the properties of the `Event` enum.